README.md

```diff
@@ -409,6 +413,88 @@ object.
     </tbody>
 </table>
 
+#### `gen-test-runner` Config
+All configuration options for the `gen-test-runner` task are found on the `config.genTestRunner`
+object.
+
+##### `GenTestRunner`
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>configs</code></td>
+            <td><code>List&lt;TestRunnerConfig&gt;</code></td>
+            <td><code>[TestRunnerConfig()]</code></td>
+            <td>The list of runner configurations used to create individual test runners</td>
+        </tr>
+    </tbody>
+</table>
+
+##### `TestRunnerConfig`
+
+<table>
+    <thead>
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>additionalImports</code></td>
```

Comment:
seems odd to have additional paths / include files specified with different naming patterns (`additionalImports` + `scriptPaths`).  Should we pick 1 pattern or the other (`additionalImports + additionalScripts` or `importPaths + scriptPaths`)?

Personally, I'd be fine with `imports` and `scripts`.  Any other opinions?